R/pkg/R/SQLContext.R

```diff
@@ -17,6 +17,34 @@
 
 # SQLcontext.R: SQLContext-driven functions
 
+#' Temporary function to reroute old S3 Method call to new
+#' We need to check the class of x to ensure it is SQLContext before dispatching
+dispatchFunc <- function(newFuncSig, x, ...) {
+  funcName <- as.character(sys.call(sys.parent())[[1]])
+  f <- get0(paste0(funcName, ".default"))
```

Comment:
I am using R 3.1.1, and find no get0 in the base package. It seems that get0 is added in a later R version? If so, either we change our supported R version, or we don't use get0 for version compatibility.